openapi/generated/lollipop_definitions.yaml

```diff
@@ -0,0 +1,108 @@
+openapi: 3.0.1
+info:
+  title: LolliPoP Definitions
+  description: The definitions of the headers needed by the LolliPoP protocol.
+  version: 1.0.0
+
+# Dummy unused endpoint
+paths:
+  /dummy:
+    get:
+      responses:
+        "200":
+          description: "ok"
+
+components:
+  parameters:
+
+    # --------------------------------------------
+    # Custom LolliPoP headers
+    # --------------------------------------------
+    LollipopMethodHeader:
+      name: x-pagopa-lollipop-original-method
+      in: header
+      description: 
+      required: true
+      schema:
+        $ref: "#/components/schemas/LollipopMethod"
+    LollipopOriginalURLHeader:
+      name: x-pagopa-lollipop-original-url
+      in: header
+      description: 
```

Comment:
Description missing. Maybe something like
The url of the original endpoint

Reply:
Done in [[#ICC-296] fix regex](https://github.com/pagopa/io-backend/pull/973/commits/0ba3a6cdaeb5d633cda84a540fe3680a41474a0a)